deployment/modules/distributor/main.tf

```diff
@@ -188,6 +223,16 @@ resource "google_cloud_run_v2_service" "default" {
         }
       }
 
+      startup_probe {
+        initial_delay_seconds = 1
+        timeout_seconds = 1
+        period_seconds = 3
+        failure_threshold = 1
```

Comment:
Does this mean it'll only do 1 probe and then shut the container down if it's not good?

Reply:
Set this and period_seconds back to the default settings as I don't have a strong opinion ahead of time on what they should be. We can tune them based on real-world experience.